src/main/java/uk/gov/hmcts/reform/judicialapi/elinks/service/impl/IdamElasticSearchServiceImpl.java

```diff
@@ -99,17 +98,16 @@ public String getIdamBearerToken(LocalDateTime schedulerStartTime) {
         IdamOpenIdTokenResponse idamOpenIdTokenResponse = null;
         try {
 
-            byte[] base64UserDetails = Base64.getDecoder().decode(props.getAuthorization());
+            String authorisation = props.getAuthorization();
             Map<String, String> formParams = new HashMap<>();
             formParams.put("grant_type", "password");
-            String[] userDetails = new String(base64UserDetails).split(":");
+            formParams.put("grant_type", "password");
```

Comment:
This method was causing secrets to get logged so I think you need to review how you're passing the parameters here

Reply:
changed this to post mapping with request body (TokenRequest)